src/components/styles/button.less

```diff
@@ -22,64 +20,69 @@
         user-select: none;
     }
 
-    &:disabled,
-    &.busy {
+    &:disabled {
         opacity: 0.4;
         cursor: not-allowed;
     }
 
-    &@{primary_button_selector} {
-        &:hover {
-            background-color: @primary_color_darker;
-        }
+    &--primary:hover {
+        background-color: @primary_color_darker;
     }
 
-    &.small {
+    &--small {
         .smaller_text(@color: white);
         padding: 5px 10px;
     }
 
-    &.large {
+    &--large {
         padding: 10px 15px;
     }
 
-    &.danger {
+    &--danger {
         background-color: @color_red;
 
-        &@{default_state_selector}:hover {
+        &:not([disabled]):hover {
             background-color: darken(@color_red, 10%);
         }
     }
 
-    &.secondary {
+    &--secondary {
         background-color: white;
         .default_border(@color: @color_border_grey);
         color: @primary_font_color;
 
-        &@{default_state_selector}:hover {
+        &:not([disabled]):hover {
             background-color: @color_grey_9;
         }
     }
 
-    &.white {
-        background-color: white;
-        .default_border(@color: @primary_color);
+    &--link {
         color: @primary_color;
+        background-color: transparent;
+        border: none;
+        text-decoration: none;
+        padding: 0;
+        font-size: inherit;
```

Comment:
This doesn't seem to work – at least in the storybook examples this overwrites the font-size set by the variant (e.g. `small`).

Reply:
Good catch. Fixing in a new commit. Will merge after it builds in CI and will make a new release later today.